main/ncurses/template.py

```diff
@@ -1,8 +1,10 @@
 pkgname = "ncurses"
-pkgver = "6.4"
-pkgrel = 2
+pkgver = "6.5"
+pkgrel = 0
 build_style = "gnu_configure"
 configure_args = [
+    "--disable-root-access",
```

Comment:
there's also root-environ but idk if that should also be disabled

Reply:
all i can find is that arch enables it and then https://github.com/kovidgoyal/kitty/issues/6842

they still enable it https://gitlab.archlinux.org/archlinux/packaging/packages/ncurses/-/blob/main/PKGBUILD?ref_type=heads#L55

Reply:
better to keep it on then :)

Reply:
debian disables root-access but not root-environ

Reply:
guess we'll match that then